tests/sam/README.md

```diff
@@ -0,0 +1,86 @@
+# AWS Serverless Application Model (AWS SAM)
+
+## Overview
+
+This folder, `tests/sam`, provides an **experimental** approach to running an instance of the Lambda locally that responds to, and returns responses like, a deployed AWS Lambda behind a Function URL, Automatic Load Balancer (ALB), or a Gateway API (all of which can be thought of as "event sources" for invoking a Lambda).
+
+This Lambda application is somewhat unique in that it also includes a CLI "sidecar" that is geared towards making HTTP requests to the deployed Lambda.  As such, for development and testing, having a local instance of the Lambda that accepts the same requests, and returns a true HTTP response, is quite helpful.
+
+## SAM Installation
+
+Ensure that AWS SAM CLI is installed: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html.
```

Comment:
Hmm, do you think you can clarify what / how user should provide the AWS credentials? 🤔 

I wasn't 100% sure how to fulfill the prerequisites outlined on this page: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html. For instance, I didn't know whether I needed to first run `aws sso login --profile-name <aws-profile-name>` in the current terminal or whether I needed to run `aws configure` first, etc...